tooling/camel-spring-boot-generator-maven-plugin/src/main/java/org/apache/camel/springboot/maven/RequiredProductizedArtifactsReader.java

```diff
@@ -50,6 +50,8 @@ public static HashMap<String, Boolean> getProductizedCSBArtifacts(File requiredF
                     map.replace(line, true);
                 }
             }
+            br.close();
```

Comment:
@cunningt I do not think the SAST analysis is happy with this fix, why can't you use try-with-resources? or close the resources in a finally block

Reply:
changed to try with resources